.evergreen/config.yml

```diff
@@ -36,60 +36,51 @@ functions:
     # Make an evergreen exapanstion file with dynamic values
     - command: shell.exec
       params:
-        working_dir: "src"
+        shell: bash
         script: |
-           echo "" > expansion.yml
-
+           pushd src
```

Comment:
its not available on all the distros. this was the first thing I did too, but got errors :(